### DISCUSSION TITLE
exporting to medium does not include visualizations

### DISCUSSION THREAD

**Discussion by kelsoufi** [datapane/datapane#181] [Help] [answered]
hello,
I am trying to export a report I wrote on datapane to medium, however only the text is being exported.
None of the table or visualizations are being exported.

**Comment by mands** [comment, marked as answer]
Sorry about the issue you're hitting, the medium exporter hasn't always worked correctly due to the Medium API changing quite often. We'll look into it a see if a fix is possible
